Configuration/Geometry/scripts/dict2023Geometry.py

```diff
@@ -437,16 +437,18 @@
     "I2" : {
         1 : [
             'Geometry/HGCalCommonData/data/fastTimingBarrel.xml',
-            'Geometry/HGCalCommonData/data/fastTiming.xml',
-            'Geometry/HGCalCommonData/data/v2/fastTimingElement.xml',
-            'Geometry/HGCalCommonData/data/fastTimingConst.xml'
+            'Geometry/HGCalCommonData/data/fastTimingEndcap.xml',
+            'Geometry/HGCalCommonData/data/fastTimingElement.xml',
             ],
         3 : [
             'Geometry/HGCalSimData/data/fasttimesens.xml'
             ],
         4 : [
             'Geometry/HGCalSimData/data/fasttimeProdCuts.xml'
             ],
+        "sim" : [
+            'from Geometry.HGCalCommonData.fastTimeParametersInitialization_cfi import *'

```

Comment:
`from Geometry.HGCalCommonData.fastTimeNumberingInitialization_cfi import *` also needs to be added